arches/management/commands/packages.py

```diff
@@ -665,8 +665,11 @@ def create_package(self, dest_dir):
 
     @staticmethod
     def update_resource_geojson_geometries():
-        with connection.cursor() as cursor:
-            cursor.execute("SELECT * FROM refresh_geojson_geometries();")
+        if not connection.in_atomic_block:
+            with connection.cursor() as cursor:
+                cursor.execute("SELECT * FROM refresh_geojson_geometries();")
+        else:
+            print("WARNING: Not refreshing geometries - transaction active")
```

Comment:
`self.stdout.write()`/`self.stderr.write()` is preferred to `print` in commands for ease of redirecting, esp. during tests

Reply:
I can change all the `print()` calls to `self.stdout.write()` as part of this PR if that makes sense.